layouts/partials/list.html

```diff
@@ -0,0 +1,143 @@
+    {{ $extractParent := $currentPage.Parent.Page }}
+    {{ $currentParent := (replaceRE "Page\\(\\/(.*)\\)" "$1" $extractParent) }}
+    <!-- Seqential Navigation for section pages. -->
+    {{ range .Children }}
+      {{ if eq $currentParent (.Parent) }}
+        {{ if .Weight }}
+          <!-- Add content to be modified by JavaScript. -->
+          {{ if $currentPage.IsMenuCurrent "docs" . }}
+            <!-- Add filler html and use JavaScript to put in correct links. -->
+            {{ partial "list-seq-nav.html" (dict "fontSize" $fontSize "sideMargin" $sideMargin "verticalMargin" $verticalMargin "currentPage" $currentPage) }}
+          {{ end }}
+        {{ else }}
+          <!-- Add content to be modified by JavaScript. -->
+          {{ if $currentPage.IsMenuCurrent "docs" . }}
+            <!-- Add filler html and use JavaScript to put in correct links. -->
+            {{ partial "list-seq-nav.html" (dict "fontSize" $fontSize "sideMargin" $sideMargin "verticalMargin" $verticalMargin "currentPage" $currentPage) }}
+          {{ end }}
+        {{ end }}
+        {{ if .HasChildren }}
+          <!-- Seqential Navigation for nested section pages. -->
+          {{ range .Children }}
+            <!-- Add content to be modified by JavaScript. -->
+            {{ if eq $currentPage (.Page) }}
+              <!-- Add filler html and use JavaScript to put in correct links. -->
+              {{ partial "list-seq-nav.html" (dict "fontSize" $fontSize "sideMargin" $sideMargin "verticalMargin" $verticalMargin "currentPage" $currentPage) }}
+            {{ end }}
+          {{ end }}
+        {{ end }}
+      {{ else }}
+        {{ if .HasChildren }}
+          <!-- Seqential Navigation for nested section pages. -->
+          {{ range .Children.ByWeight }}
+            <!-- Add content to be modified by JavaScript. -->
+            {{ if $currentPage.IsMenuCurrent "docs" . }}
+              <!-- Add filler html and use JavaScript to put in correct links. -->
+              {{ partial "list-seq-nav.html" (dict "fontSize" $fontSize "sideMargin" $sideMargin "verticalMargin" $verticalMargin "currentPage" $currentPage) }}
+            {{ end }}
+          {{ end }}
+        {{ else }}
+          <!-- Add content to be modified by JavaScript. -->
+          {{ if $currentPage.IsMenuCurrent "docs" . }}
+            <!-- Add filler html and use JavaScript to put in correct links. -->
+            {{ partial "list-seq-nav.html" (dict "fontSize" $fontSize "sideMargin" $sideMargin "verticalMargin" $verticalMargin "currentPage" $currentPage) }}
+          {{ end }}        
+        {{ end }}
+      {{ end }}
+    {{ end }}  
+  {{ end }}
+{{ end }}
+
+<script>
+ // IMPORTANT NOTE - this depends on HTML attributes to work.
+ // If error, then link href will be "javascript:void(0)" 
+ // and text output of links will render as:
+ // <- Page   Page ->
+ 
+ // DEPENDENT VARIABLES - redefine if HTML attributes or semantics change.

```

Comment:
Variables in case dependent HTML tags and/or attributes change. These could be implemented elsewhere though.